### DISCUSSION TITLE
Time Keeping

### DISCUSSION THREAD

**Discussion by Pspritechologist** [FakeFishGames/Barotrauma#8057] [Ideas and suggestions]
My suggestion is extremely simple: A clock.
Or rather, a couple of clocks. I feel, personally, that the introduction of a ticking clock does a substantial amount to make a world feel more "lived in," and in a world like Barotrauma's, where time is almost entirely unimportant to the life of the people around, it would be as easy as setting a variable that ticks up in some multiple of real-time, presumably measured out so your average mission lasts a few hours, and preferably displayed in a 24 hour format (or an 85 hour format, the length of an actual day on Europa).

Two items would be more than enough; an analog clock that simply ticks around, and a clock component with a singe output to send the current time. A digital clock would be easy to make using the component and a text display.

My personal primary use for this would be book keeping, where I wanted to track when people entered the armory onboard my sub in a terminal, logging down when it occurred.

**Comment by MaaiG** [comment]
Ship’s time. +1
